Core/PixelEvent.swift

```diff
@@ -320,10 +320,13 @@ extension Pixel {
         case adAttributionLogicWrongVendorOnSuccessfulCompilation
         case adAttributionLogicWrongVendorOnFailedCompilation
         
-        case debugBookmarkOrphanFolder
-        case debugBookmarkTopLevelMissing
-        case debugFavoriteOrphanFolder
-        case debugFavoriteTopLevelMissing
+        // 2 suffix added to differenciate from previous pixels when the fix wasn't available
+        case debugBookmarkOrphanFolder2
+        case debugBookmarkTopLevelMissing2
+        case debugFavoriteOrphanFolder2
+        case debugFavoriteTopLevelMissing2
+        case debugCouldNotFixBookmarkFolder
+        case debugCouldNotFixFavoriteFolder
```

Comment:
I don't love the 2 suffix, I'd rather have something more meaningful (esp since when looking at pixels themselves we won't have the context of the comment)

Reply:
@THISISDINOSAUR  any suggestion?
maybe debugFavoriteFixOprhanFolder / debugBookmarkFixOprhanFolder?

Reply:
I'd maybe just replace the 2 with new, which should give a reasonable hint that the one without any further context is old